pages/index.js

```diff
@@ -1,43 +1,53 @@
-// import external resources
-import useLocalStorageState from "use-local-storage-state";
+import { useState } from "react";
 import styled from "styled-components";
-// import internal resources
-import ProjectList from "@/components/ProjectList";
-import Logo from "@/components/svg/Logo.svg";
-import ProjectForm from "@/components/ProjectForm";
+import Logo from "@/public/Logo.svg";
 
-export default function HomePage({
-  addProject,
-  projects,
-  handleDeleteProject,
-}) {
+export default function HomePage({}) {
+  const [isClicked, setIsClicked] = useState(false);
+
+  function handleClick() {
+    setIsClicked(!isClicked);
+  }
   return (
     <>
-      <Header>
-        <Title>my projects</Title>
-      </Header>
-
-      <ProjectList
-        projects={projects}
-        handleDeleteProject={handleDeleteProject}
-      />
-
-      <ProjectForm addProject={addProject} />
+      <PageWrapper>
+        <Wrapper>
+          <Logo />
+          <a
+            href={`/ProjectPage`}
+            style={{ textDecoration: "none", color: "#94c3dd" }}
+          >
+            <StyledDiv onClick={handleClick} isClicked={isClicked}>
+              Welcome
+            </StyledDiv>
+          </a>
```

Comment:
You should work with next/link and not with an anchor tag as a normal anchor tag results in a reload of your app. [Read more about next/link in the next.js docs.](https://nextjs.org/docs/api-reference/next/link)
You can then style the next/link directly or work with the example provided in the next.js docs for styled-components (see the mentioned link).